Formula/zellij.rb

```diff
@@ -0,0 +1,24 @@
+class Zellij < Formula
+  desc "Pluggable terminal workspace, with terminal multiplexer as the base feature"
+  homepage "https://zellij.dev"
+  url "https://github.com/zellij-org/zellij/archive/refs/tags/v0.8.0.tar.gz"
+  sha256 "850d1a68219debdb4e05f7f58e70b31f440425247caaa1b29b9be1b6c7869207"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+
+    ohai "Proper status bar rendering requires nerdfont to be installed and used in the terminal."
```

Comment:
Users won't ever see this unless they build from source. This should be in the upstream documentation though, unless it's brew specific.

Reply:
This is documented in [upstream](https://zellij.dev/documentation/compatibility.html#the-status-bar-fonts-dont-render-correctly).

Reply:
Yes they have this documented in upstream, however I thought it was a good idea to let users know about it. Should I make this a caveat section ?

Reply:
No, because it is not a homebrew specific issue.